### ISSUE TITLE
Feature request : initialize form with global values

### ISSUE THREAD

**Issue by benoitverstraete** [danvick/flutter_form_builder#47]
Hi @danvick,

Is it possible to load initial values in the form only once instead of setting all field's "initialValue" manually ? 

Maybe something like this :

`_formBuilderKey.currentState.value = <String, dynamic>{ 'firstName': 'Initial value'};`

Or in a new property (`FormBuilder.initialValues` for example).

Thanks a lot for your help and the latest improvements in the package !

**Comment by danvick** [comment]
It's a good idea that I've been considering but I think it will be planned for after verios 3.0.0 release. May be a duplicate of [#31](https://github.com/danvick/flutter_form_builder/issues/31)

**Comment by danvick** [comment]
Hi @benoitverstraete,
As of version 3.3.0, one can set the initial value for the entire form.
